### ISSUE TITLE
Can I just download the .bin and drop it on SD card or do I need to update the config file and complile.. I have the ZYLTech Gear v3

### ISSUE THREAD

**Issue by galluccib** [d3dorsett/Marlin-2.x-for-ZYLTech-Gear-BLTouch#1]

**Comment by d3dorsett** [comment]
Yes, you can copy just the .bin file to the SD card and power on the printer. It will install. Run a few simple homing test, before printing to make sure it works correctly. Babysteps in the control panel may need to be set once the nozzle height is where it needs to be.

**Comment by galluccib** [comment]
Got it printed and firmware loaded!!

Everything seems to be working and I’m stuck on setting the “Z-Offset”.

I keep changing it in -> Bed Leveling - > Edit Mesh -> Z Value = but for some reason something keep over writing it. This is the correct place to be setting it right?

Sorry for the stupid questions just getting started with 3D printing…

**Comment by galluccib** [comment]
Quick question… I pulled down your config files and the latest Marlin and changed “Baud rate to 128 so I can use Cura and enabled the “PROBE_OFFSET_WIZARD” in adv and when I go to build it I get

Compiling .pio\build\mks_robin_nano35\src\src\HAL\STM32F1\eeprom_flash.cpp.o
In file included from Marlin\src\HAL\STM32F1\../../inc/MarlinConfig.h:49:0,
                 from Marlin\src\HAL\STM32F1\HAL.cpp:30:
Marlin\src\HAL\STM32F1\../../inc/SanityCheck.h:70:4: error: #error "You are using an old Configuration.h file, update it before building Marlin."

   #error "You are using an old Configuration.h file, update it before building Marlin."

Pull down 2.0.7.2

https://github.com/MarlinFirmware/Marlin/releases

When you did the build do you recall what version you used?

Thanks,

From: Brian Gallucci
Sent: Sunday, January 17, 2021 11:15 PM
To: d3dorsett/Marlin-2.x-for-ZYLTech-Gear-BLTouch <reply@reply.github.com>; d3dorsett/Marlin-2.x-for-ZYLTech-Gear-BLTouch <Marlin-2.x-for-ZYLTech-Gear-BLTouch@noreply.github.com>
Subject: RE: [d3dorsett/Marlin-2.x-for-ZYLTech-Gear-BLTouch] Can I just download the .bin and drop it on SD card or do I need to update the config file and complile.. I have the ZYLTech Gear v3 (#1)

Got it printed and firmware loaded!!

Everything seems to be working and I’m stuck on setting the “Z-Offset”.

I keep changing it in -> Bed Leveling - > Edit Mesh -> Z Value = but for some reason something keep over writing it. This is the correct place to be setting it right?

Sorry for the stupid questions just getting started with 3D printing…
